index.html

```diff
@@ -4129,95 +4106,81 @@ <h3>Favor Abstract Claims</h3>
       </p>
     </section>
 
-    <section class="informative">
-      <h3>The Principle of Data Minimization</h3>
+    <section class="informative" id="the-principle-of-data-minimization">
+      <h3>데이터 최소화 원칙</h3>
 
       <p>
-Privacy violations occur when information divulged in one context leaks into
-another. Accepted best practice for preventing such violations is to limit the
-information requested, and received, to the absolute minimum necessary. This
-data minimization approach is required by regulation in multiple jurisdictions,
-including the Health Insurance Portability and Accountability Act (HIPAA) in the
-United States and the General Data Protection Regulation (GDPR) in the European
-Union.
+한 컨텍스트에서 유출된 정보가 다른 컨텍스트로 유출되면 개인정보보호 위반이 발생한다. 
+이러한 위반을 방지하기 위해 허용되는 모범 사례는 요청 및 수신한 정보를 최소한으로 
+제한하는 것 이다. 데이터 최소화 접근법은 미국의 
+HIPAA (Health Insurance Portability and Accountability Act) 및 EU의 
+General Data Protection Regulation (GDPR)을 포함한 여러 관할권의 규정에 의해 
```

Comment:
```suggestion
GDPR (General Data Protection Regulation)을 포함한 여러 관할권의 규정에 의해 
```
HIPAA와 표현 형식을 맞추는 것이 좋을 것 같습니다.